### ISSUE TITLE
Disabling showing currency symbol

### ISSUE THREAD

**Issue by bcersows** [aguirrel/ng-currency#23]
Hey, thanks for your module, it's cool.

Quick question, do you plan on adding an option to disable the currency symbol? I am using Bootstrap's input groups for showing the currency, so having it twice is kind of weird.

-edit-
As some suggestions, nice would be
- parameter to not show any currency symbol at all (no-currency-symbol)
- parameter to hide currency symbol when clicking into the input field (hide-currency-symbol)

-/edit-

Thanks for your response,
bcersows

**Comment by aguirrel** [comment]
Hi @bcersows,

You can use currency-symbol="" option to disable currency symbol.  It works fine and do both things. ;)

Please, if you need further assistance reopen this ticket.

Thanks!

**Comment by bcersows** [comment]
Hey,

nice idea. That solves the problem, even though it leaves a single space after the value. But that seems ok for me.

Closing the ticket, thanks for the hint!

bcersows

**Comment by bcersows** [comment]
Hey.

Sorry to bother again, but I found a rather weird behaviour.
I tried applying currency-symbol to the inputs.
Most worked, except on one page.
![githubngcurrencyissue](https://cloud.githubusercontent.com/assets/1199637/7413726/6311abde-ef4e-11e4-99b9-6cb46b859ac0.png)
As seen in the image, initially the currency symbol is shown. Only after changing the value, it is hidden.

The exact same markup
```
<input ng-model="cost.Amount"
								name="costAmount"
								type="text"
								ng-currency min="0" currency-symbol=""
								class="form-control" />
```
works on all the other pages.
Might there be a difference because those inputs are inside of a ngRepeat?

Both pages get their data from a server, so I do not suspect the issue there; inspecting the GET with FireBug, I see nothing wrong.
Just for documentation, the first block is the sample response that does get iterated over, but where the symbol is shown, the second one is the working data.
```
[{"Id":43,"YearId":0,"Caption":"test","Amount":0.4},{"Id":44,"YearId":0,"Caption":"testt","Amount":7
.0}]
```
```
{"Id":1012,"Caption":"lala","PricePerUsage":0.0,"PricePerMonth":0.1,"ProjectId"
:2018}
```
Obviously, "Amount", "PricePerUsage" and "PricePerMonth" are the money values.

Have you got any idea?
Thanks.
